spec/views/players/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "players/create.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
same, pending spec where the test or why we need them?

Reply:
same answer)